packages/deployment/app/cloudformation.template.yml

```diff
@@ -410,11 +410,9 @@ Resources:
       Namespace: AWS/ES
       Dimensions:
         - Name: ClientId
-          Value:
-            Fn::Sub: ${AWS::AccountId}
+          Value: !Sub ${AWS::AccountId}
```

Comment:
Is there some reason for changing to use intrinsic functions? The cloudformation template isn't using them anywhere else that I can see, so I think we're intentionally not using them. This may have something to do with Kes YAML merging. 

All things equal, it seems better to stick the the `Fn::Sub` syntax

Reply:
ok